quickwit/quickwit-serve/src/index_api/rest_handler.rs

```diff
@@ -183,6 +269,36 @@ async fn create_index(
     index_service.create_index(index_config, false).await
 }
 
+fn clear_index_handler(
+    index_service: Arc<IndexService>,
+) -> impl Filter<Extract = (impl warp::Reply,), Error = Rejection> + Clone {
+    warp::path!("indexes" / String / "clear")
+        .and(warp::post())
```

Comment:
`warp::delete()` ?

Reply:
we don't delete the index, `PUT` is more appropriate I guess.